tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 3.5.3
-envlist = flake8,py{37,36,27}-ansible{27,26,25,devel}
+envlist = flake8,py{37,36,35,27}-ansible{27,26,25,devel}
```

Comment:
travis config would also have to be modified to test this env if we go this road.